### PR TITLE
#3133 - fixed image block description when no url present

### PR DESCRIPTION
**Pull request by Mathiu**
## Description
#3133 fix - BlockDescription was missing from InspectorControls when no url was present.

## How Has This Been Tested?
Firefox 58 beta 6

## Types of changes
Simple fix.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

**Review by jorgefilipecosta — APPROVED** [review]
Thank you for your contribution @Mathiu. I tested the changes and things look fine to me 👍 
In my tests it looks like this change also solves the problem reported in https://github.com/WordPress/gutenberg/issues/3658.
I left a minor suggestion but this looks ready to ship :) @Mathiu Could you squash the changes in one commit?

File: blocks/library/image/block.js [hunk 1]
```diff
@@ -190,9 +201,7 @@ class ImageBlock extends Component {
 			controls,
 			focus && (
 				<InspectorControls key="inspector">
-					<BlockDescription>
-						<p>{ __( 'Worth a thousand words.' ) }</p>
-					</BlockDescription>
+					{blockDescription}
```
Comment (jorgefilipecosta) [inline blocks/library/image/block.js]: Minor: Changing {blockDescription} to { blockDescription } make the change more in line with the codebase. The Gallery block currently misses the spaces and will be corrected.

**Comment by Mathiu** [comment]
OK, i have no idea how to squash commits together. I have tried and I think this: https://github.com/Mathiu/gutenberg/commit/aceeb49c9edf726565b07d84e54ea5403ff36886 is the what squashed commit should be, but i have no idea if you can merge this commit alone.

Sorry, I have never done that before and I'm new to any kind of contribution. Thank you for your time and patience.

**Comment by Mathiu** [comment]
I think i broke this PR so i made a new one with requested changes and a single commit: #3756.

Sorry again for the mess.

**Comment by jorgefilipecosta** [comment]
Closing this one as a new PR https://github.com/WordPress/gutenberg/pull/3756 was created. Thank you for creating a new PR with the requested changed @Mathiu.

**Pull request closed without merging**